galaxy_ng/tests/integration/api/rbac_actions/utils.py

```diff
@@ -112,46 +113,32 @@ def wait_for_task(resp, path=None, timeout=300):
     return resp
 
 
-def podman_login(username, password):
-    cmd = [
-        "podman",
-        "login",
-        "--username",
-        f"{username}",
-        "--password",
-        f"{password}",
-        "localhost:5001",
-        TLS_VERIFY
-    ]
+def ensure_test_container_is_pulled():
+    cmd = ["podman", "container", "exists", TEST_CONTAINER]
     proc = Popen(cmd, stdout=PIPE, stderr=STDOUT, encoding="utf-8")
```

Comment:
These `Popen`s might be a good candidates for `subprocess.run()` if you just want to call the command and wait, but maybe not if you want the test to fall back to `alpine` if it cannot reach `TEST_CONTAINER`

Reply:
switched to subprocess.run()